docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: "3.2"
+
+services:
+  api-gateway:
+    image: itl-teams/api-gateway:latest
+    build: ./api/example
+    container_name: api-gateway
+    restart: unless-stopped
+    ports:
+      - "80:80"
+
```

Comment:
### Note
This gateway is taking the example config, I'm going to change this by myself anyways.
Just keep in mind that.